insights/specs/default.py

```diff
@@ -287,7 +287,7 @@ def docker_container_ids(broker):
     hammer_ping = simple_command("/usr/bin/hammer ping")
     hammer_task_list = simple_command("/usr/bin/hammer --csv task list")
     haproxy_cfg = first_file(["/var/lib/config-data/puppet-generated/haproxy/etc/haproxy/haproxy.cfg", "/etc/haproxy/haproxy.cfg"])
-    heat_api_log = first_file(["/var/log/containers/heat/heat-api.log", "/var/log/heat/heat-api.log"])
+    heat_api_log = first_file(["/var/log/containers/heat/heat_api.log", "/var/log/heat/heat_api.log"])
```

Comment:
For OpenStack-10, I think the name is `heat-api.log`, (see 
[1.1.11. Orchestration (heat) Log Files](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/10/html-single/logging_monitoring_and_troubleshooting_guide/index#orchestration_heat_log_files) and [Orchestration log files](https://docs.openstack.org/mitaka/config-reference/orchestration/logs.html)), the `head_api.log` is from OpenStack13(?) from your investigation result. And Insights should support all versions of OpenStack from 8 to 14. So, both the `heat_api.log` and `heat-api.log` should be collected.

In that case, just like @psachin mentioned in https://github.com/RedHatInsights/insights-core/issues/1886#issuecomment-489515790, you should add the items of `heat_api.log` to the spec `heat_api_log`, but should not remove the original items. e.g.

```python
heat_api_log = first_file(["/var/log/containers/heat/heat-api.log", "/var/log/heat/heat-api.log", "/var/log/containers/heat/heat_api.log", "/var/log/heat/heat_api.log"])
```

I'm not familiar with OpenStack, if there is anything wrong, please correct me freely.

Reply:
@xiangce 
I have OSP 12,13,14 setup in which the openstack services are deployed with containers.  I have checked and heat api logs can be seen as heat_api.log and not heat-api.log. So in containerized deployment it is /var/log/containers/heat/heat_api.log where as for non-containerized deployment such as OSP 10 it is /var/log/heat/heat-api.log. 
So in the spec we can add the containerized path as var/log/containers/heat/heat_api.log as the log path may have changed
heat_api_log = first_file(["/var/log/containers/heat/heat_api.log", "/var/log/heat/heat-api.log"])